store/tikv/coprocessor.go

```diff
@@ -473,6 +475,10 @@ func (it *copIterator) open(ctx context.Context) {
 			respChan: it.respChan,
 			finishCh: it.finishCh,
 			vars:     it.vars,
+			clientHelper: clientHelper{
+				LockResolver:        it.store.lockResolver,
+				RegionRequestSender: sender,
```

Comment:
maybe take care `RegionRequestSender` is  a stateful object, better not share it between different request scops

https://github.com/pingcap/tidb/blob/095bee9655520f8c26cff5fb6569909292906f07/store/tikv/region_request.go#L60 this need be request scope(introduced by reviwer coocood :running_man: ) to mark this request has be access all peers in regions and all of them are failed,  at last trigger reload.